File: src/main/java/com/procurement/contracting/ContractingApplication.java
package com.procurement.contracting;

import com.procurement.contracting.config.ApplicationConfig;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

@SpringBootApplication(scanBasePackageClasses = ApplicationConfig.class)
@EnableEurekaClient
public class ContractingApplication {
    public static void main(final String[] args) {
        SpringApplication.run(ContractingApplication.class, args);
    }
}

File: src/main/kotlin/com/procurement/contracting/ContractingApplication.kt
package com.procurement.contracting

import com.procurement.contracting.infrastructure.configuration.ApplicationConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackageClasses = [ApplicationConfig::class])
class ContractingApplication

fun main(args: Array<String>) {
    runApplication<ContractingApplication>(*args)
}